app/views/govuk_wrapper.scala.html

```diff
@@ -54,6 +58,14 @@
     <meta name="format-detection" content="telephone=no" />
 }
 
+@scriptElem = {
+    <script src="@routes.Assets.at("javascripts/scripts.js")" type="text/javascript"></script>
```

Comment:
Please remove this scripts.js since it's `direct-debit-frontend` specific stuff for accessibility and not related to timeout dialogs.